src/lib/loaded/Loaded.svelte

```diff
@@ -0,0 +1,9 @@
+<script lang="ts">
```

Comment:
I feel like this should be called `Mounted.svelte` for `onMount`.

Reply:
`Mounted.svelte` sounds good to me!

Reply:
I tried it out, and found that `OnMount` made the most sense.  I kinda like it.  What do you think?  @ghostdevv hbu?

Reply:
Not fussed.